### ISSUE TITLE
Unexpected behaviour with conditional parameter selectors using custom populating queries

### ISSUE THREAD

**Issue by nielsdejong** [neo4j-labs/neodash#369]
Consider this scenario:
- Two parameter selector reports are created.
- The first one is a simple node property selector (e.g. select $neodash_person_name)
- The second one uses a custom query to populate the results based on the value of $neodash_person_name. For example, to let the selector only choose movies that the person has acted in:
```
MATCH (p:Person)-[:ACTED_IN]->(m:Movie)
WHERE toLower(toString(m.`title`)) CONTAINS toLower($input)
AND  p.name = $neodash_person_name
RETURN m.title as value LIMIT 5
```

Then, the conditional query works for the first time it is created, but after the $neodash_person_name value changes, it doesn't populate results any more. Suspected is that this related to the advanced setting to clear parameters on field reset.

**Comment by GQ16** [comment]
Hey @nielsdejong , how do I even make the custom parameter selector like you've described:
"The second one uses a custom query to populate the results based on the value of $neodash_person_name. For example, to let the selector only choose movies that the person has acted in:"
Where do I even go in the settings of the Parameter Select widget to do that?

**Comment by BennuFire** [comment]
Solved with 2.2.4, no? @nielsdejong
